### ISSUE TITLE
IE 8 and IE 9 not supported

### ISSUE THREAD

**Issue by pmorch** [bug7a/progressbar.js#1]
Hi,

I've noticed that IE9 and IE8 aren't supported. A mention of what browsers are supported would go a long way to help determine for folks if this is a usable option for them.

Feel free to just close or ignore if you disagree.

Peter

**Comment by rocklan** [comment]
Absolutely, please put this on your page. Doesn't work on IE9 because classList is unsupported, and also because dispatchEvent() isn't supported.
